### ISSUE TITLE
Just a loading screen

### ISSUE THREAD

**Issue by JoshAlvarado** [nethruster/ptemplate#21]
when I view it on a live server it's just a loading screen.

**Comment by claudio4** [comment]
Can you check if there is any error message in the browser's console or in the webpack terminal?

**Comment by JoshAlvarado** [comment]
https://pastebin.com/mMR8aSDd
this is what i get after entering npm install and npm build dev

**Comment by gariasf** [comment]
Hi @JoshAlvarado, I've updated the project dependencies and the Webpack configuration so it'll work with more recent Webpack versions.

Please pull the latest commit from master and try again. Everything should work correctly.

I appreciate your patience.

**Comment by JoshAlvarado** [comment]
Before i close the issue how where are the setting for the contact me form?

**Comment by gariasf** [comment]
You need to use [this tool](https://github.com/claudio4/webformTelegramHook) we made.

`formUrl` in the config.js file should point to it and when properly configured messages will be sent to you by the telegram bot.

Let us know if you have further questions.

**Comment by JoshAlvarado** [comment]
For users who don't use telegram is there a way to get notification through email? Is telegram the only form?

**Comment by Miguel-Dorta** [comment]
That tool was a quick fix aimed primarily to our teams' personal use. In theory, ptemplate should work fine with any backend that have the same API, but in practice it's practically impossible to find any backend that have this, specially considering that there are no standard API for this kind of forms, and the fact that ptemplate's API is not documented (looking at you - @gariasf and @claudio4 - this is a #todo).

I've suggested to the team that we should adapt ptemplate to use a proper solution to the backend problem, insisting in using [this](https://github.com/Miguel-Dorta/web-msg-handler) which is already fully developed and have a very similar API.

We'll inform you about this. Thank you for your patience.

**Comment by gariasf** [comment]
@JoshAlvarado 

Since the sending email process is actually part of the backend side of the webapp, we didn't want to assume systems or environments. There are plenty of tools on the internet to send emails, so we just gather the information from the form and send it via HTTP.

To be precise, what really happens is: 

We're sending, via HTTP POST, an object of the following format to whatever is listening on the `formUrl` endpoint (specified in the config):
```json
{
  "name": "John Doe",
  "email": "johndoe@example.com",
  "message": "Hello world",
  "g-recaptcha-response": "alphanumericrandomstring"
}
```
Any tool capable of receiving that HTTP call can do anything with the information, including sending emails.

If you feel like you want to code it yourself, go ahead, it's lots of fun.
However, we also think that we should make a "recommended" tool for the job. So in the meantime, we'll try to come up with a solution ourselves.

**Comment by mneonwill** [comment]
I'm closing this issue due to inactivity.

**Comment by Miguel-Dorta** [comment]
> I'm closing this issue due to inactivity.

You closed this to inactivity just a day after we fixed it 🤦

Anyway, for the reference: It was **fixed** in 04e607f2a66810d3b9bb34f00a966163fd983401. It is now documented in the README.md

**Comment by mneonwill** [comment]
> > I'm closing this issue due to inactivity.
> 
> You closed this to inactivity just a day after we fixed it 🤦
> 
> Anyway, for the reference: It was **fixed** in 04e607f2a66810d3b9bb34f00a966163fd983401. It is now documented in the README.md

Thank you Miguel, I wasn't aware of that. 

In the future,  it is  perhaps a good idea to update an issue as soon as it is fixed.
